templates/security/disclosure-policy.html

```diff
@@ -0,0 +1,124 @@
+{% extends "templates/one-column.html" %}
+
+{% block title %}Ubuntu Security disclosure and embargo policy{% endblock %}
+
+{% block meta_description %}Canonical and the Ubuntu Security Team participate in responsible disclosure and collaborate with the wider community on security issues. This describes how to contact the Ubuntu Security Team, what you can expect when you contact us, and what we expect from you.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1WdamniXOHemuPzl6vESYD9StPgh4-PepbqdVHYrzbuI/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-8 u-vertically-center">
+      <h1>
+        Ubuntu Security disclosure and embargo policy
+      </h1>
+      <p>
+        <i>Valid since: Oct-2020</i>
+      </p>
+      <p>
+        Canonical and the Ubuntu Security Team participate in responsible disclosure and collaborate with the wider community on security issues. This describes how to contact the Ubuntu Security Team, what you can expect when you contact us, and what we expect from you.
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      <h2>
+        About Canonical
+      </h2>
+      <p>
+        Canonical publishes the Ubuntu operating system in collaboration with a community of Ubuntu developers. Canonical also publishes other software projects such as LXD, MAAS, Juju, snapd, Snapcraft, Landscape, Launchpad and Mir.
+      </p>
+      <p>
+        Canonical's Ubuntu Security Team tends to the security needs of the Ubuntu operating system and serves as a point of contact for Canonical-authored software, both proprietary and open-source, as well as Canonical-owned and -managed infrastructure.
+      </p>
+      <p>
+        Please contact us if you believe you have found a security issue in Ubuntu, Canonical software or Canonical services.
+      </p>
+      <h2>
+        How to report an issue to us
+      </h2>
+      <p>
+        You may report issues to the Ubuntu Security Team via the Launchpad.net bug reporting interface ("ubuntu-bug &ltpackagename&gt" is the most convenient way to get to the bug reporting form). Please be aware that Launchpad.net will send email in plaintext in response to bug reports.
+      </p>
+      <p>
+        You may also send email to <a href="mailto:security@ubuntu.com">security@ubuntu.com</a>. Email may optionally be encrypted to OpenPGP key "4072 60F7 616E CE4D 9D12 4627 98E9 740D C345 39E0": https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x407260f7616ece4d9d12462798e9740dc34539e0
```

Comment:
```suggestion
        You may also send email to <a href="mailto:security@ubuntu.com">security@ubuntu.com</a>. Email may optionally be encrypted to OpenPGP key <code>4072 60F7 616E CE4D 9D12 4627 98E9 740D C345 39E0": https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x407260f7616ece4d9d12462798e9740dc34539e0</code>
```

this should be a code block